src/ReportIssue/ReportIssueForm.tsx

```diff
@@ -16,7 +16,7 @@ import { SvgXml } from "react-native-svg"
 
 import { useVersionInfo } from "../hooks/useApplicationInfo"
 import { GlobalText, Button } from "../components"
-import { reportAnIssue, ReportIssueError } from "../More/zendeskAPI"
+import { reportAnIssue, ReportIssueError } from "../Settings/zendeskAPI"
```

Comment:
Should this api file be moved to this folder? Is it used anywhere else?

Reply:
Basically just renaming the More folder to Settings so I think this is ok for now!